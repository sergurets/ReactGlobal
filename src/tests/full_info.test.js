import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Full_info from '../components/full_info';

const data = [
    {
      budget:null,
      genres:[],
      overview:'overview',
      poster_path:'poster_path',
      release_date:"release_date",
      runtime:'runtime',
      title:'title',
    }      
  ]


describe('full_info', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Full_info
        overview={data.overview} 
        title={data.title} 
        runtime={data.runtime}
        release_date={data.release_date}  
        poster_path={data.poster_path}/>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
