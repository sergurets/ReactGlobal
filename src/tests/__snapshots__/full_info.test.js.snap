// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`full_info should render correctly 1`] = `
<div
  className="full_info"
>
  <div
    className="full_info_controls"
  >
    <h2>
      Netflix search
    </h2>
    <button
      className="searchButton"
    >
      Search
    </button>
  </div>
  <div
    className="full_info_wrapper"
  >
    <img />
    <div
      className="full_info_content"
    >
      <h1 />
      <div
        className="content_data"
      >
        <span
          className=""
        />
      </div>
      <p
        className=""
      />
    </div>
  </div>
</div>
`;
